### ISSUE TITLE
[BUG] Social icons are blocked by uBlock Origin

### ISSUE THREAD

**Issue by SwampertX** [dillonzq/LoveIt#343]
### Describe the bug 描述你遇到的错误
Social icons are not showing in Firefox. Shows properly in Chrom(ium).

### Expected behavior 期待的行为
Social icons should show in Firefox too.

### Screenshots 屏幕截图
![image](https://user-images.githubusercontent.com/17807016/81554684-ce718380-93b9-11ea-85ff-0edbe6a7abbe.png)
Firefox (75 and 76)
![image](https://user-images.githubusercontent.com/17807016/81554876-24dec200-93ba-11ea-98de-4cafc9dcc47a.png)
Inspector in Firefox
![image](https://user-images.githubusercontent.com/17807016/81554942-3cb64600-93ba-11ea-9fa7-5e898d4b1442.png)
Chromium
如果可以的话, 提供屏幕截图对解决问题很有帮助.

### Build Environment 构建环境

- OS: Ubuntu 20.04
- Theme version/commit v0.2.6
- Hugo version `Hugo Static Site Generator v0.71.0-DEV/extended linux/amd64 BuildDate: unknown`

### Preview Environment 预览环境

- OS: Ubuntu 20.04
- Browser: Firefox 75, 76 (Fails to display social icons), Chromium 81.0 (Success)

### Additional Information 补充信息
```
[params.home.profile]
  social = true
[params.social]
  # relevant entries, non empty
```

**Comment by dillonzq** [comment]
Hello, @SwampertX
Have you modified `assets/data/social.yml`?

**Comment by SwampertX** [comment]
Hello, no I have not. I merely changed the values in the inspector to see if that works, but turns out that it didn't work (before and after I added the `fa` class).

**Comment by dillonzq** [comment]
Hi, @SwampertX
You don't need to add `fa` for FontAwesome 5.
Maybe FontAwesome CSS is not loaded correctly?

**Comment by SwampertX** [comment]
It is loaded correctly. The icons show well everywhere else except for in the `<div class="social-icons"></div>`. Could you reproduce it?

**Comment by dillonzq** [comment]
@SwampertX Sorry, I can't reproduce it.
Could you see social icons of https://hugoloveit.com on your Firefox browser?

**Comment by SwampertX** [comment]
Found the issue. It is [uBlock Origin](https://getublock.com/) that is blocking the social links. I think it just disables divs with class names containing `social`, preventing them from loading at all. Changing the `profile.html` partials and the `_home.scss` into, say `.external-icons` works for me now.

> Could you see social icons of https://hugoloveit.com on your Firefox browser?

Without disabling uBlock Origin, I also could not see the icons even on https://hugoloveit.com.

As uBlock Origin is a pretty popular add-on, do you have any plans to edit the class name for the social links? If not it is fine, I could make it work for my case.

**Comment by dillonzq** [comment]
> Found the issue. It is [uBlock Origin](https://getublock.com/) that is blocking the social links. I think it just disables divs with class names containing `social`, preventing them from loading at all. Changing the `profile.html` partials and the `_home.scss` into, say `.external-icons` works for me now.
> 
> > Could you see social icons of [hugoloveit.com](https://hugoloveit.com) on your Firefox browser?
> 
> Without disabling uBlock Origin, I also could not see the icons even on [hugoloveit.com](https://hugoloveit.com).
> 
> As uBlock Origin is a pretty popular add-on, do you have any plans to edit the class name for the social links? If not it is fine, I could make it work for my case.

Great work! I would consider an appropriate solution.

**Comment by dillonzq** [comment]
Fixed in #354
